src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-hadoop-monitoring-integration.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: Apache Hadoop integration
```

Comment:
i think our general pattern is that we like 'monitoring' in the titles of these, for SEO value